src/databricks/labs/ucx/hive_metastore/table_migration_status.py

```diff
@@ -171,7 +191,10 @@ def _iter_catalogs(self) -> Iterable[CatalogInfo]:
 
     def _iter_schemas(self) -> Iterable[SchemaInfo]:
         for catalog in self._iter_catalogs():
-            if catalog.name is None:
+            if catalog.name is None or catalog.catalog_type in (
```

Comment:
This has probably the same effect as the filter on line 186. If not, move the filter there so that filtering catalogs happens inside the `_iter_catalog`